ansible_collections/arista/avd/roles/eos_designs/doc/node-types.md

```diff
@@ -107,12 +107,12 @@ vtep_ip:
 - `{{ switch_id }}`
 - `{{ loopback_ipv4_offset }}`
 
-`p2p_uplinks_ip` & `p2p_uplinks_peer_ip` can leverage the switch facts (switch.*) to customize the IP adressing.
-All other templates can only leverage the input variables and variables set in switch facts template.
+While all templates can leverage the internal switch facts (switch.*) to customize the interface descriptions,
```

Comment:
It varies a bit. We need to change our templates, to use the `switch.*` values instead of some temporary variables. But that would break any customer templates, so we need to maintain these until next major version.
So I don't want to spend too much time highlighting all the legacy variables. I think I will change the current templates, and update the docs to suggest only using the `switch.*` facts (which are still subject to change). Existing deployments using these vars will still work because of the extra care we take in the template functions.